pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[build-system]
```

Comment:
Hasn't this been addressed in the previous PR? I'm assuming that this is just duplicated here

Reply:
It was just to make it easier to merge the pyproject.toml first.